modules/ztp-deploying-siteconfig-hub-side-template.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-deploying-siteconfig-hub-side-template_{context}"]
+= Deploying a SiteConfig with hub-side templating
+
+The following is an example `SiteConfig` custom resource (CR) that uses the `spec.clusters.siteConfigMap.data` field to specify per-site data to be used in hub side templating.
+
+.Example {sno} `SiteConfig` CR with hub-side templating
+[source,yaml]
+----
+include::snippets/ztp_example-siteconfig-hub-side-templating.yaml[]
+----
+[NOTE]
```

Comment:
Maybe the section below can be a bit restructured to something like:

>Maybe all the section above can be a bit restructured to something like:
Besides the `data` field, the `siteConfigMap` section also includes the namespace where the `ConfigMap` is to be created and its name. 

For a successful integration with the PolicyGenTemplate, the `siteConfigMap.namespace` you specify must be the same as the namespace used by the `PolicyGenTemplate` resources.

If not specified, `siteConfigMap.namespace` will default to <ztp-site>, and `siteConfigMap.namespace` to ztp-site-<cluster_name>.

Reply:
Maybe we could have another look at this together?